vividus-plugin-mobile-app/src/main/java/org/vividus/selenium/mobileapp/screenshot/MobileAppAshotFactory.java

```diff
@@ -47,24 +54,31 @@ public MobileAppAshotFactory(ScreenshotCropper screenshotCropper, MobileAppWebDr
     }
 
     @Override
+    @SuppressFBWarnings("SF_SWITCH_FALLTHROUGH")
     public AShot create(Optional<ScreenshotParameters> screenshotParameters)
     {
         String strategyName = screenshotParameters.flatMap(ScreenshotParameters::getShootingStrategy)
                 .orElseGet(this::getScreenshotShootingStrategy);
-
         ShootingStrategy strategy;
-        if (appendBottomNavigationBarOnAndroid && mobileAppWebDriverManager.isAndroid())
-        {
-            strategy = getStrategyBy(strategyName).getDecoratedShootingStrategy(ShootingStrategies.simple());
-            int statusBarSize = mobileAppWebDriverManager.getStatusBarSize();
-            strategy = decorateWithFixedCutStrategy(strategy, statusBarSize, 0);
-        }
-        else
+        switch (strategyName)
         {
-            strategy = getStrategyBy(strategyName).getDecoratedShootingStrategy(new MobileViewportShootingStrategy());
-            strategy = downscale ? scaling(strategy, (float) this.getDpr()) : strategy;
+            case SIMPLE:
```

Comment:
Can we make `getStrategyBy(strategyName)` return appropriate strategy?

Reply:
Not possible now, as there dependencies on type of mobile OS